Gameheads-SAP-2024/Assets/Scripts/Managers/spriteFlicker.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class spriteFlicker : MonoBehaviour
+{
+
+    public GameObject flickerIcon;
```

Comment:
This is cool, I'd add [SerializeField] to front of this so we can assign it a gameobject in the inspector (Like for the continue button!)